src/backend/services/workflow_service.py

```
@@ -303,15 +303,18 @@ def run_agentic_workflow(natural_language_query: str, model_provider: str, model
                 temp_storage = get_temp_metrics_storage()
                 browser_metrics = temp_storage.read_browser_metrics(workflow_id) or {}
                 logging.info(f"📊 Browser-use metrics: {browser_metrics}")
+                logging.info(f"📊 DEBUG: browser_metrics tokens = {browser_metrics.get('tokens', 'NOT_FOUND')}")
```

Ruff (0.14.10)

`info()` call on root logger

(LOG015)